.claude/scripts/mcp_common.sh

```
@@ -0,0 +1,2130 @@
+export MEMORY_FILE_PATH="\$HOME/.cache/mcp-memory/memory.json"
+export ${debug_env_var}=false
+export MCP_VERBOSE_TOOLS=false
+export MCP_AUTO_DISCOVER=false
+exec "$NPX_PATH" @modelcontextprotocol/server-memory "\$@"
+EOF
+    chmod +x "$WRAPPER_SCRIPT"
+
+    # Add server using the wrapper script
+    capture_command_output fallback_output fallback_exit_code "${MCP_CLI_BIN}" mcp add "${MCP_SCOPE_ARGS[@]}" "${DEFAULT_MCP_ENV_FLAGS[@]}" "memory-server" "$WRAPPER_SCRIPT"
+
+    if [ $fallback_exit_code -eq 0 ]; then
```

Shellcheck (0.11.0)

[warning] fallback_exit_code is referenced but not assigned.

(SC2154)